src/naive/time/tests.rs

```diff
@@ -265,7 +266,7 @@ fn test_date_from_str() {
     assert!("".parse::<NaiveTime>().is_err());
     assert!("x".parse::<NaiveTime>().is_err());
     assert!("15".parse::<NaiveTime>().is_err());
-    assert!("15:8".parse::<NaiveTime>().is_err());
+    assert!("15:".parse::<NaiveTime>().is_err());
```

Comment:
Why just add test assert `"15:"` ? (I think assert `"15:8"` was doing something useful here)

Reply:
Instead of `15:8` in the invalid cases we have the similar ` 9:08` in the valid cases.
`15:` is new, to check we don't allow a trailig colon.

Reply:
The `15:8` exercises combination double-digit hour, single-digit minute
whereas ` 9:08` is exercising single-digit hour leading space, double-digit minute leading zero.
I recommend keeping `15:8`.